tests/test_integral/perfume.spec.ts

```diff
@@ -0,0 +1,365 @@
+    });
+    describe('# without Token Test', () => {
+        it('# getPerfume_success', (done: Done) => {
+            request(app)
+                .get(`${basePath}/perfume/1`)
+                .expect((res: request.Response) => {
+                    expect(res.status).to.be.eq(StatusCode.OK);
+                    const result: ResponseDTO<PerfumeDetailResponse> = res.body;
+                    expect(result.message).to.be.eq(
+                        MSG_GET_PERFUME_DETAIL_SUCCESS
+                    );
+                    result.data.volumeAndPrice.forEach(
+                        (volumeAndPrice: string) => {
+                            expect(volumeAndPrice).to.be.not.contain('NaN');
+                        }
+                    );
+                    done();
+                })
+                .catch((err: Error) => done(err));
+        });
+        describe('# searchPerfume Test', () => {
+            const tests: any[] = [
+                {
+                    args: {
+                        searchText: 'Tom',
+                        keywordList: [],
+                        ingredientList: [],
+                        brandList: [],
+                    },
+                    expected: 0,
+                },
+                {
+                    args: {
+                        searchText: 'Chanel',
+                        keywordList: [],
+                        ingredientList: [],
+                        brandList: [],
+                    },
+                    expected: 62,
+                },
+            ];
+            var i = 0;
+            tests.forEach(
+                ({ args, expected }: { args: any; expected: any }) => {
+                    it(`searchPerfumeTest case ${i++}`, (done: Done) => {
+                        request(app)
+                            .post(`${basePath}/perfume/search`)
+                            .send(args)
+                            .expect((res: request.Response) => {
+                                expect(res.status).to.be.eq(StatusCode.OK);
+                                const result: ResponseDTO<
+                                    ListAndCountDTO<PerfumeResponse>
+                                > = res.body;
+                                expect(result.message).to.be.eq(
+                                    MSG_GET_SEARCH_PERFUME_SUCCESS
+                                );
+                                expect(result.data!!.count).to.be.gte(expected);
+                                done();
+                            })
+                            .catch((err: Error) => done(err));
+                    });
+                }
+            );
+        });
+        describe('# getNewPerfume Test', () => {
+            it('success case', (done: Done) => {
+                request(app)
+                    .get(`${basePath}/perfume/new`)
+                    .expect((res: request.Response) => {
+                        expect(res.status).to.be.eq(200);
```

Comment:
```suggestion
                        expect(res.status).to.be.eq(StatusCode.OK);
```

status code 명시하는 방식을 통일하면 좋을것 같습니다.